src/Microsoft.AspNetCore.Routing/EndpointOptions.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.AspNetCore.Routing
 {
-    public class DispatcherOptions
+    public class EndpointOptions
```

Comment:
I'm still not 100% happy with our options for what to do with this. It doesn't seem to me like adding the data sources to `RoutingOptions` is a great thing to do... So I did something simple for now. If anyone has an amazing suggestion I'm looking for ideas.